tests/foreman/cli/test_classparameters.py

```diff
@@ -31,68 +31,54 @@
 from robottelo.config import settings
 from robottelo.constants.repos import CUSTOM_PUPPET_REPO
 from robottelo.datafactory import gen_string
-from robottelo.test import CLITestCase
 
 
-@pytest.mark.run_in_one_thread
-class SmartClassParametersTestCase(CLITestCase):
-    """Implements Smart Class Parameter tests in CLI"""
+@pytest.fixture(scope='module')
+def module_org():
+    return make_org()
 
-    @classmethod
-    @pytest.mark.skipif((not settings.repos_hosting_url), reason='Missing repos_hosting_url')
-    def setUpClass(cls):
-        """Import some parametrized puppet classes. This is required to make
-        sure that we have smart class variable available.
-        Read all available smart class parameters for imported puppet class to
-        be able to work with unique entity for each specific test.
-        """
-        super().setUpClass()
-        cls.puppet_modules = [{'author': 'robottelo', 'name': 'cli_test_classparameters'}]
-        cls.org = make_org()
-        cls.loc = make_location()
-        cv = publish_puppet_module(cls.puppet_modules, CUSTOM_PUPPET_REPO, cls.org['id'])
-        cls.env = Environment.list({'search': 'content_view="{}"'.format(cv['name'])})[0]
-        Environment.update(
-            {
-                'name': cls.env['name'],
-                'organization-ids': cls.org['id'],
-                'location-ids': cls.loc['id'],
-            }
-        )
-        cls.puppet_class = Puppet.info(
-            {'name': cls.puppet_modules[0]['name'], 'environment': cls.env['name']}
-        )
-        cls.sc_params_list = SmartClassParameter.list(
-            {
-                'environment': cls.env['name'],
-                'search': 'puppetclass="{}"'.format(cls.puppet_class['name']),
-            }
-        )
-        cls.sc_params_ids_list = [sc_param['id'] for sc_param in cls.sc_params_list]
-        cls.host = entities.Host(
-            organization=cls.org['id'], location=cls.loc['id'], environment=cls.env['name']
-        ).create()
-        cls.host.add_puppetclass(data={'puppetclass_id': cls.puppet_class['id']})
-        cls.hostgroup = make_hostgroup(
-            {'environment-id': cls.env['id'], 'puppet-class-ids': cls.puppet_class['id']}
-        )
 
-    @classmethod
-    def tearDownClass(cls):
-        """Removes puppet class."""
-        super().tearDownClass()
-        delete_puppet_class(cls.puppet_class['name'])
+@pytest.fixture(scope='module')
+def module_loc():
+    return make_location()
```

Comment:
Same suggestion, but the global fixture is called `module_location`

Reply:
ack